doc/oper.xml

```diff
@@ -1647,43 +1647,52 @@ true
 <ManSection>
   <Oper Name="IsMatching" Arg="digraph, list"/>
   <Oper Name="IsMaximalMatching" Arg="digraph, list"/>
+  <Oper Name="IsMaximumMatching" Arg="digraph, list"/>
   <Oper Name="IsPerfectMatching" Arg="digraph, list"/>
   <Returns><K>true</K> or <K>false</K>.</Returns>
   <Description>
     If <A>digraph</A> is a digraph and <A>list</A> is a list of pairs of
     vertices of <A>digraph</A>, then <C>IsMatching</C> returns <K>true</K> if
     <A>list</A> is a matching of <A>digraph</A>.  The operations
-    <C>IsMaximalMatching</C> and <C>IsPerfectMatching</C> return <K>true</K> if
-    <A>list</A> is a maximal, or perfect, matching of <A>digraph</A>,
-    respectively.  Otherwise, these operations return <K>false</K>. <P/>
+    <C>IsMaximalMatching</C>, <C>IsMaximumMatching</C> and <C>IsPerfectMatching</C> 
+    return <K>true</K> if <A>list</A> is a maximal, maximum or perfect, matching 
+    of <A>digraph</A>, respectively.  Otherwise, these operations return <K>false</K>.
+    <P/>
 
     A <E>matching</E> <C>M</C> of a digraph <A>digraph</A> is a subset of the
     edges of <A>digraph</A>, i.e. <C>DigraphEdges(<A>digraph</A>)</C>, such
     that no pair of distinct edges in <C>M</C> are incident to the same vertex
     of <A>digraph</A>.  Note that this definition allows a matching to contain
     loops.  See <Ref Prop="DigraphHasLoops" />.  The matching <C>M</C> is
-    <E>maximal</E> if it is contained in no larger matching of the digraph, and
+    <E>maximal</E> if it is contained in no larger matching of the digraph,
+    is <E>maximum</E> if it has the greatest cardinality among all matchings and
     is <E>perfect</E> if every vertex of the digraph is incident to an edge in
-    the matching.  Every perfect matching is maximal.
+    the matching.  Every maximum or perfect matching is maximal. Note however
```

Comment:
Maybe
"Note, however, that for digraphs with loops, not every perfect matching will be maximum."
or
"Note, however, that not every perfect matching of digraphs with loops is maximum."

Reply:
Also this is just a bit upsetting in general, but if that's how it's already defined then not sure what can be done about it